exporters/trace/jaeger/agent.go

```diff
@@ -103,15 +105,66 @@ func newAgentClientUDP(params agentClientUDPParams) (*agentClientUDP, error) {
 	}
 
 	return &agentClientUDP{
-		connUDP:       connUDP,
-		client:        client,
-		maxPacketSize: params.MaxPacketSize,
-		thriftBuffer:  thriftBuffer,
+		connUDP:        connUDP,
+		client:         client,
+		maxPacketSize:  params.MaxPacketSize,
+		thriftBuffer:   thriftBuffer,
+		thriftProtocol: thriftProtocol,
 	}, nil
 }
 
 // EmitBatch implements EmitBatch() of Agent interface
```

Comment:
Not related, but this would be a helpful inclusion.

```suggestion
// EmitBatch buffers batch to fit into UDP packets and sends the data to the agent.
```